src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const usedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const isUsedThis = tab => {
```

Comment:
```suggestion
  const handleSelect = tab => {
```
Just FYI: such an example of name is a bit better
`isUsedThis ` sounds like it is a boolean variable or at least a function that **returns** boolean variable